app/api/dao/user.py

```diff
@@ -142,13 +143,14 @@ def get_user_by_username(username):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id, search_query="", is_verified=None):
+    def list_users(user_id, search_query='', is_verified=None, sort_order=''):
```

Comment:
Why is sort_order not None by default?

Reply:
> Why is sort_order not None by default?

@ramitsawhney27 here sort_order is ' ' empty string, if 1 or -1 is passed then, sort accordingly as ascending or descending order else default behaviour will be taken.